### PR TITLE
Data and Methods (deadline: Wed)

### PR DESCRIPTION
**Pull request by diegocasmo**
+ data section
  - [x] description of the problem in more detail (Diego)
  - [x] discuss data attributes (features, labels, etc) (Diego)

+ methods
  - [ ] Introduction to deep learning(Ankur)
  - [x] Introduction to CNN (Diego)
  - [ ] Choice of architecture ``VGG16`` (Tristan)

### CODE REVIEW

**Review by diegocasmo — COMMENTED** [review]
### Deep learning section review (@ankurshukla03):
- first paragraph ``features or factors or concepts`` reads a little weird, I would suggest only using the word features (since we already use it in other sections of the report)
- neural networks description should come first than deep learning (deep learning as we use it here is a subset of neural networks)
  - I would actually suggest splitting this section in to two: neural networks, and deep learning
- second paragraph: ``Some neurons may influence the environment by triggering actions.\cite{NeuralNetwork}`` where does that citation belong? is it from the paragraph before the period, or after it?
- unanswered questions/issues:
  - how are the neurons connected?
  - what is an activation function?
  - how does the network learn? (this looks like a good place to introduce supervised learning)
  - why is the credit assignment a problem?
  - there's no mention anywhere of gradient descent/backpropagation

**Review by diegocasmo — COMMENTED** [review]
### CNN section review (@diegocasmo):
- unanswered questions/issues:
  - [x] what is the input layer (CNNs assume input are images)
  - [x] missing details on feature maps:
    - depth
    - stride
    - zero-padding
  - [x] what about ReLU?
    - the filter computes element-wise multiplication and addition (linear), the ReLU layer adds non-linearity
    - why do we need to add non-linearity?
  - [x] how does the pooling layer work? (there's no mention of max-pooling, average pooling, etc)
  - [x] how is a CNN trained? (here's a good [overview](https://ujjwalkarn.me/2016/08/11/intuitive-explanation-convnets/))

**Review by tristaaan — APPROVED** [review]
cool stuff, open an issue for Ankur and Tristan on their respective sections
